pkg/did/creator/creator_test.go

```diff
@@ -11,84 +11,131 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/trustbloc/wallet-sdk/pkg/localkms"
```

Comment:
move this line down

Reply:
Done in https://github.com/trustbloc/wallet-sdk/pull/52